dll/win32/msgina/gui.c

```diff
@@ -282,6 +310,18 @@ StatusDialogProc(
             DlgData_Destroy(hwndDlg);
             return TRUE;
         }
+
+        case WM_PAINT:
```

Comment:
Why is it now painted manually instead of just adding the image to the control?

Reply:
The Normal and VGA bitmap are two separate Bitmaps. It is painted manually so it can display the correct bitmap.

Reply:
You can just activate the correct bitmap in the control using `STM_SETIMAGE` or something like that?

Reply:
From what I've seen throughout this code, this is exactly how the Bitmap is painted on each dialog.